### PR TITLE
Bobby javavscript modernization

### PR DESCRIPTION
**Pull request by bobbyt1997**

### CODE REVIEW

File: cadence/server/public/js/aria.js [hunk 1]
```diff
@@ -1,190 +1,146 @@
...
+   getListenURL()
+   getNowPlayingMetadata()
+   getNowPlayingAlbumArt()
+   getVersion()
+   connectRadioData()
+   postSearch()
+   postRequestID()
+})
+
+let getVersion = () => {
```
Comment (za419) [inline cadence/server/public/js/aria.js:23]: Why is this a `let`? I don't think we should be reassigning these functions.

(etc for the others)

File: cadence/server/public/js/page.js [hunk 2]
```diff
@@ -1,51 +1,49 @@
...
+      stream.src = ''
+      stream.load()
+      stream.pause()
+      document.getElementById(PLAY_BUTTON).innerHTML = '⏵'
+   }
+
+   // Loads and unloads audio stream source
+   document
+      .getElementById(PLAY_BUTTON)
+      .addEventListener('click', () => stream.paused ? play() : pause())
```
Comment (za419) [inline cadence/server/public/js/page.js:31]: Should we refactor this to be jQuery?

```js
$(`#${PLAY_BUTTON}`).click(() => stream.paused ? play() : pause())
```

Comment (za419) [reply cadence/server/public/js/page.js:31]: Also goes for other uses of `document.getElementBy*`

**Comment by kenellorando** [comment]
@bobbyt1997

**Pull request closed without merging**